### PR TITLE
Release 2022-07-17

### PR DESCRIPTION
**Pull request by palma21**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+  * The [NodeRestriction Admission Controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#noderestriction) will be enabled.  This will allow users to enable/disable node restriction.
+  * CoreDNS version 1.9.2 will be default version. With this new version of CoreDNS wildcard queries are no longer allowed.
+  * metrics-server version 0.6.1 will be the default version.
+  * metrics-server vertical pod autoscaler will be enabled.
+* Kubernetes 1.21 version deprecation will start taking effect from July 31st, 2022. 
+* Update your AKS labels to the recommended substitutions before deprecation after the Kubernetes v1.24 release. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* Konnectivity rollout is finished in global and started in Sovereign (China, USGov).
+
```
Comment (allyford) [inline CHANGELOG.md:18]: Docker will no longer be supported as a container runtime after September 1, 2022. Follow these [steps](https://docs.microsoft.com/en-us/azure/aks/learn/quick-windows-container-deploy-cli#:~:text=Upgrade%20an%20existing%20Windows%20Server%20node%20pool%20to%20containerd) in our documentation to upgrade your Kubernetes cluster to change your container runtime to containerd.

**Comment by olsenme** [comment]
[Azure Blob CSI Driver](https://github.com/kubernetes-sigs/blob-csi-driver#set-up-csi-driver-on-aks-cluster-only-for-aks-users) is now supported in public preview in AKS. Follow these [instructions](https://docs.microsoft.com/en-us/azure/aks/azure-blob-csi?tabs=NFS) to use blob csi driver as a managed addon to mount blob storage to a pod via blobfuse or NFS 3.0 options.

File: CHANGELOG.md (hunk 1 above)
Comment (palma21) [reply CHANGELOG.md:18]: thanks, remember to remove location from the doc links :)

**Review by qpetraroia — APPROVED** [review]
My announcements look good, ty!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,64 @@
...
+  * CoreDNS version 1.9.2 will be default version. With this new version of CoreDNS wildcard queries are no longer allowed.
+  * metrics-server version 0.6.1 will be the default version.
+  * metrics-server vertical pod autoscaler will be enabled.
+* Kubernetes 1.21 version deprecation will start taking effect from July 31st, 2022.
+* Update your AKS labels to the recommended substitutions before deprecation after the Kubernetes v1.24 release. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* Konnectivity rollout is finished in global and started in Sovereign (China, USGov).
+* Docker will no longer be supported as a container runtime on Windows after September 1, 2022. Follow these [steps](https://docs.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli#:~:text=Upgrade%20an%20existing%20Windows%20Server%20node%20pool%20to%20containerd) in our documentation to upgrade your Kubernetes cluster to change your container runtime to containerd.
+
+
+## Release notes
```
Comment (shashankbarsin) [inline CHANGELOG.md:21]: nit: should it be 3 #

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,64 @@
...
+  * CoreDNS version 1.9.2 will be default version. With this new version of CoreDNS wildcard queries are no longer allowed.
+  * metrics-server version 0.6.1 will be the default version.
+  * metrics-server vertical pod autoscaler will be enabled.
+* Kubernetes 1.21 version deprecation will start taking effect from July 31st, 2022.
+* Update your AKS labels to the recommended substitutions before deprecation after the Kubernetes v1.24 release. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* Konnectivity rollout is finished in global and started in Sovereign (China, USGov).
+* Docker will no longer be supported as a container runtime on Windows after September 1, 2022. Follow these [steps](https://docs.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli#:~:text=Upgrade%20an%20existing%20Windows%20Server%20node%20pool%20to%20containerd) in our documentation to upgrade your Kubernetes cluster to change your container runtime to containerd.
+
+## Release notes
```
Comment (shashankbarsin) [inline CHANGELOG.md:48]: same as above. should it be 3 #?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,64 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2022-07-24
+
```
Comment (shashankbarsin) [inline CHANGELOG.md:4]: should we add estimated time for rollout completion?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,64 @@
...
+* Docker will no longer be supported as a container runtime on Windows after September 1, 2022. Follow these [steps](https://docs.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli#:~:text=Upgrade%20an%20existing%20Windows%20Server%20node%20pool%20to%20containerd) in our documentation to upgrade your Kubernetes cluster to change your container runtime to containerd.
+
+## Release notes
+
+* Preview Features
+  * KEDA Addon is now supported on ARM64-based nodes.
+  * [Azure Blob CSI Driver](https://github.com/kubernetes-sigs/blob-csi-driver#set-up-csi-driver-on-aks-cluster-only-for-aks-users) is now supported in public preview in AKS. Follow these [instructions](https://docs.microsoft.com/azure/aks/azure-blob-csi?tabs=NFS) to use blob csi driver as a managed addon to mount blob storage to a pod via blobfuse or NFS 3.0 options.
+* Features
+  * The annotation `kubernetes.azure.com/set-kube-service-host-fqdn` can now be added to pods to set the KUBERNETES_SERVICE_HOST variable to the domain name of the API server instead of the in-cluster service IP. This is useful in cases where the cluster egress is via a layer 7 firewall, like Azure Firewall with Application Rules.
```
Comment (shashankbarsin) [inline CHANGELOG.md:54]: documentation link?

File: CHANGELOG.md (hunk 4 above)
Comment (palma21) [reply CHANGELOG.md:4]: No, that's what the tracker is for

File: CHANGELOG.md (hunk 5 above)
Comment (palma21) [reply CHANGELOG.md:54]: There isn't one AFAIK

Comment (palma21) [reply CHANGELOG.md:54]: @phealy should we add to https://docs.microsoft.com/en-us/azure/aks/limit-egress-traffic ?
on the API server IP sections

**Pull request merged**